frontend/src/components/Navbar/Navbar.js

```diff
@@ -1,12 +1,9 @@
-import React, {useState} from 'react'
+import React from 'react'
 import {Bars, Nav, NavLink, NavMenu} from "./NavBarElements";
 
 
 export default function Navbar() {
 
-    const [isOpen, setOpen] = useState(false)
-
-
     return (
         <>
             <Nav>
```

Comment:
`activeStyle` in `NavLink` is supposed to be an object with styles? 🤔 https://reactrouter.com/web/api/NavLink